lib/DDG/Meta/Data.pm

```diff
@@ -164,8 +164,8 @@ sub get_ia {
 sub get_js {
     my ($self, $id) = @_;
 
-    my $metaj = encode_json($self->get_ia(id => $id));                                                                                                                                                                                              
-	return qq(DDH.$id = DDH.$id || {};\nDDH.$id.meta = $metaj;); 
+    my $metaj = eval { encode_json($self->get_ia(id => $id)) } or warn "Failed to encode metdata to json: $@";
```

Comment:
I'm just trying to prevent a naive dereference in JavaScript. So whatever the top level object is should probably be what `$metaj` is set to in the event of an error.